### ISSUE TITLE
Python built in modules are not being detected in vscode.

### ISSUE THREAD

**Issue by Yiannis128** [microsoft/pyright#6524]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

If you have a question about typing or a behavior that you’re seeing in Pyright (as opposed to a bug report or enhancement request), consider posting to the [Pyright discussion forum](https://github.com/microsoft/pyright/discussions).

**Describe the bug**

Typehints such as `NamedTuple` and annotations like `@dataclass` are getting `reportUndefinedVariable` red lines. They have already been imported and successfully run.

**Code or Screenshots**

![image](https://github.com/microsoft/pyright/assets/9535618/4fb8a513-ae30-4a56-8025-77695548fc56)

**VS Code extension or command-line**

Pyright extension in vscodium on fedora 39 linux.

```
VSCodium Version: 1.84.2
Pyright Version: v1.1.337
```

**Comment by erictraut** [comment]
It looks like you're using a notebook. Pyright doesn't contain support for notebooks. Pyright is focused on type checking. It contains basic support for language server features, but these features are limited.

The Pylance language server, which is built on top of pyright, adds additional functionality including notebook support. Pylance isn't licensed for use with VSCodium, so if you want to use this functionality, you'd need to switch to VS Code.

**Comment by Yiannis128** [comment]
I see, thank you for the clarification.
